src/relay/backend/contrib/codegen_c/codegen_c.h

```diff
@@ -207,17 +215,20 @@ class CodegenCBase {
    *
    * \return The emitted code string.
    */
-  std::string JitImpl(std::string ext_func_id, std::vector<std::string> args,
+  std::string JitImpl(std::string ext_func_id, Array<Var> args,
```

Comment:
please also use const reference here for all parameters.

Reply:
Updated